spec/models/change_note_migrator_spec.rb

```diff
@@ -0,0 +1,86 @@
+require "rails_helper"
+
+RSpec.describe ChangeNoteMigrator do
+  let(:publishing_api) { double(:publishing_api) }
+  let(:guide) { create(:guide) }
+  let(:major_edition) { create(:edition, :published, update_type: "major", version: 2, guide: guide) }
+  let(:minor_edition) { create(:edition, :published, update_type: "minor", version: 2, guide: guide) }
+  let(:unpublished_edition) { create(:edition, :draft, guide: guide) }
+  let(:change_note) { "I am a change note" }
+  subject { described_class.new(publishing_api: publishing_api, dry_run: false) }
```

Comment:
Part of me thinks this could use some tests for dry run behaviour. Part of me thinks this might only be used once… up to you.